pyproject.toml

```diff
@@ -62,22 +62,15 @@ where = ["src"]
 
 [tool.pytest.ini_options]
 asdf_schema_root = 'resources/schemas'
+asdf_schema_tests_enabled = 'true'
 asdf_schema_skip_tests = """
-    stsci.edu/asdf/asdf-schema-1.0.0.yaml
-    stsci.edu/asdf/wcs/celestial_frame-1.0.0.yaml
-    stsci.edu/asdf/wcs/celestial_frame-1.1.0.yaml
-    stsci.edu/asdf/wcs/frame-1.0.0.yaml
-    stsci.edu/asdf/wcs/frame-1.1.0.yaml
-    stsci.edu/asdf/wcs/spectral_frame-1.1.0.yaml
     stsci.edu/asdf/wcs/step-1.1.0.yaml
```

Comment:
There are a few remaining skips for schemas that aren't referenced by any core schemas. However the changes in this PR allow removal of more than half the skips.